### PR TITLE
Nik/130 fix user model routes and aggregate

### PR DESCRIPTION
**Pull request by nikvijay07**
## 

Issue Number(s): #130 

What does this PR change and why?

Fix Models
Add
totalPatients, activeVolunteers, inactiveVolunteers
to the Chapter Model. Make these all of the numbers type.

Add aggregator function and onChangeFunctions
make a route in /api/chapter/aggregate
make this a Post Request.
Here, use a mongoDB query to find the total number of active volunteers, inactive volunteers, and patients associated with each chapter and update the chapter documents accordingly.

Additionally
Every time we add a new patient or volunteer, or patch a volunteer with a chapter parameter, or active parameter, we want to modify the associated chapter document. Modify the previous routes to do this.

(YET to do front end)

### Checklist

- [ ] Requirements have been implemented
- [ ] Acceptance criteria are met
- [ ] Database schema docs have been updated or are not necessary
- [ ] Code follows design and style guidelines
- [ ] Commits follow guidelines (concise, squashed, etc)
- [ ] Relevant reviewers (Senior Dev/EM/Designers) have been assigned to this PR

### Critical Changes

- Database change/migration to run
- Environment config change
- Breaking API change

### Related PRs

- #130 

### Testing

Enumerate steps to test the functionality of your ticket. This should include edge cases and testing of error handling, if applicable.

### CODE REVIEW

File: src/app/api/chapter/create-chapter/route.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  yearFounded: number;
+  country: string;
+  city?: string;
+  state?: string;
+};
+
+export const POST = APIWrapper({
+  config: {
+    //   requireToken: true,
+    //   requireAdmin: true
```
Comment (nikvijay07) [inline src/app/api/chapter/create-chapter/route.ts]: wasn't too sure what u wanted in the config for create chapter

**Review by johannesq23 — APPROVED** [review]
